solidity/test/TestPhasedEscrow.js

```diff
@@ -102,6 +103,40 @@ describe("PhasedEscrow", () => {
     })
   })
 
+  describe("withdrawFromEscrow", async () => {
+    const amount = web3.utils.toBN(12090) 
+    let escrow  
+
+    beforeEach(async () => {
+      escrow = await Escrow.new(token.address, {from: owner})
+      await token.transfer(escrow.address, amount, {from: owner})
+    })
+
+    it("pulls all funds from a non-phased Escrow when having no tokens", async () => {
+      const balanceBefore = await token.balanceOf(phasedEscrow.address)
+      expect(balanceBefore).to.eq.BN(0)
+
+      await escrow.setBeneficiary(phasedEscrow.address, {from: owner})
+      await phasedEscrow.withdrawFromEscrow(escrow.address)
+
+      const balanceAfter = await token.balanceOf(phasedEscrow.address)
+      expect(balanceAfter).to.eq.BN(amount)
+    })
+
+    it("pulls all funds from a non-phased Escrow when having some tokens", async () => {
+      const initialFunds = web3.utils.toBN(999)
+      await token.transfer(phasedEscrow.address, 999, {from: owner})
```

Comment:
I think you can pass `initialFunds` instead of `999`

Reply:
Yeah, my bad. Fixed in 09f9529.